3.10/analyzers.md

```diff
@@ -141,11 +158,39 @@ Analyzer  /  Capability                   | Tokenization | Stemming | Normalizat
 [`text`](#text)                           |     Yes      |   Yes    |     Yes       | (Yes)
 [`segmentation`](#segmentation)           |     Yes      |    No    |     Yes       |   No
 
-Available normalizations are case conversion and accent removal
-(conversion of characters with diacritical marks to the base characters).
+The available normalizations are case conversion and accents/diacritics removal.
+The `segmentation` Analyzer only supports case conversion.
 
 The `text` Analyzer supports edge _n_-grams but not full _n_-grams.
 
+Tokenization 
+------------
+
+The `text` and `segmentation` Analyzer types tokenize text into words (or a
+comparable concept of a word). See
+[Word Boundaries](https://www.unicode.org/reports/tr29/#Word_Boundaries){:target="_blank"}
+in the Unicode Standard Annex #29 about Unicode text segmentation for details.
+
+These tokenizing Analyzers extract tokens, which removes characters like
+punctuation and whitespace. An exception is the [`segmentation` Analyzer](#segmentation)
+if you select `"graphic"` or `"all"` for the `break` option. They preserve `@`
+and `.` characters of email addresses, for instance. There are also exceptions
+with both Analyzer types for sequences like numbers, for which decimal and
+thousands separators (`.` and `,`) are preserved.
+
+Normalization
+-------------
+
+The `norm`, `text`, and `segmentation` Analyzer types allow you to convert the
+input text to all lowercase or all uppercase for normalization purposes, namely
+case insensitive search. Case folding is not supported. Also see
+[Case Mapping](https://unicode-org.github.io/icu/userguide/transforms/casemappings.html){:target="_blank"}
+in the ICU documentation.
+
+The `norm` and `text` Analyzer types also allow you to convert characters with
+diacritical marks to the base characters. This normalization enables
+accent-insensitive search.
+
```

Comment:
Are there plans to describe Stemming and n-grams  here?

Reply:
Out-of-scope for this PR but I added it to the list of https://arangodb.atlassian.net/browse/DOC-242